### ISSUE TITLE
Support custom output of @prisma/client.

### ISSUE THREAD

**Issue by asmimo** [MichalLytek/typegraphql-prisma#184]
```prisma
generator client {
  provider = "prisma-client-js"
  output   = "../../node_modules/.prisma/client/mod1"
}

generator typegraphql {
  provider = "typegraphql-prisma"
  output   = "../../node_modules/.prisma/type-graphql/mod1"
}
```

Looking at the generated code prisma is imported from `@prisma/client`. I changed it manually and it works. While generating code, point the prisma import to the output directory if provided.

Edit:
And one more thing. Every resolver is exposed in graphql server.

```ts
import {} from ".prisma/type-graphql/mod1"
```

Even if i write import statement and import nothing, every resolver is automatically added in my graphql server. I am using `mercurius` for graphql server

**Environment (please complete the following information):**

- OS: Manjaro 
- Node: 16.8.0
- `typegraphql-prisma` version: 0.16.1 
- Prisma version: 3.0.1
- TypeScript version: 4.4.3

**Comment by asmimo** [comment]
I found out couple of things.
1) If prisma and `typegraphql-prisma` output is generated outside of `node_modules`, the generated code will point to the right prisma client.

2) About all resolvers exposed, it was my bad. I was providing path of other custom resolvers. I found out this in the docs of [type-graphql](https://typegraphql.com/docs/bootstrap.html)

> Be aware that in case of providing paths to resolvers files, TypeGraphQL will emit all the operations and types that are imported in the resolvers files or their dependencies.

**Comment by MichalLytek** [comment]
@asmimo Your request is against a change that I've made some time ago.
Basically, now there's a check if `prismaClientPath.includes("node_modules")`, then it uses `"@prisma/client"` as the import path.

That was introduced because there were plenty of reports about wrong path for mono-repos with yarn workspaces, which requires using the `"@prisma/client"` in order to properly resolve the deps moved to root level.

So your request says "ignore node_modules, always use provided output". It's not so simple because by default Prisma has a default output path, so I can't guess when it's the custom path to custom node_modules and when I should use the standard import 😕

**Comment by MichalLytek** [comment]
Does Prisma support multiple schemas with multiple databases in a single project? 
Because that's your issue I suppose, you want to use concrete version of Prisma Client 🤔

**Comment by asmimo** [comment]
No, Prisma doesn't support multiple schemas/databases but there is a workaround for that mentioned in this issue's [comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118). I am using the second workaround like this.

```prisma
// prisma/one/schema.prisma
generator client {
  provider = "prisma-client-js"
  output   = "../../node_modules/.prisma/client/one"
}
generator typegraphql {
  provider = "typegraphql-prisma"
  output   = "../../node_modules/.prisma/type-graphql/one"
}
...

// prisma/two/schema.prisma
generator client {
  provider = "prisma-client-js"
  output   = "../../node_modules/.prisma/client/two"
}
generator typegraphql {
  provider = "typegraphql-prisma"
  output   = "../../node_modules/.prisma/type-graphql/two"
}
...
```

I put each schema in separate folder otherwise it puts migration of both schema on single migration folder. Then run cli targeting the schema.


> Your request is against a change that I've made some time ago. Basically, now there's a check if `prismaClientPath.includes("node_modules")`, then it uses `"@prisma/client"` as the import path.

This check causes error when using like this because there is multiple generated clients on non-default prisma folder.
The workaround I did is outputting the generated prisma and typegraphql on the same schema folder.
```prisma
// prisma/one/schema.prisma
generator client {
  provider = "prisma-client-js"
  output   = "./client"
}
generator typegraphql {
  provider = "typegraphql-prisma"
  output   = "./type-graphql"
}
...
```
This works but with commonjs only. Previously I was using esm, got error and converted the project back to commonjs.

**Comment by MichalLytek** [comment]
Fixed via a016f7c 🔒
